Add error info when 'unable to init db'

Currently, I get this error, without any useful information:
```
panic: localpg: failed to start PgSQL: unable to init database using: /tmp/go_embedded_pgsql_binaries_3773798719/bin/initdb -A password -U username -D /tmp/go_embedded_pgsql_runtime_2105005878/data --pwfile=/tmp/go_embedded_pgsql_runtime_2105005878/pwfile --locale=en_US.UTF-8
```

--- a/prepare_database.go
+++ b/prepare_database.go
@@ -44,12 +44,12 @@ func defaultInitDatabase(binaryExtractLocation, runtimePath, pgDataDir, username
 	postgresInitDBProcess.Stderr = logger
 	postgresInitDBProcess.Stdout = logger
 
-	if err := postgresInitDBProcess.Run(); err != nil {
-		return fmt.Errorf("unable to init database using: %s", postgresInitDBProcess.String())
+	if err = postgresInitDBProcess.Run(); err != nil {
+		return fmt.Errorf("unable to init database using '%s': %w", postgresInitDBProcess.String(), err)
 	}
 
 	if err = os.Remove(passwordFile); err != nil {
-		return fmt.Errorf("unable to remove password file '%v': %v", passwordFile, err)
+		return fmt.Errorf("unable to remove password file '%v': %w", passwordFile, err)
 	}
 
 	return nil

--- a/prepare_database_test.go
+++ b/prepare_database_test.go
@@ -40,7 +40,7 @@ func Test_defaultInitDatabase_ErrorWhenCannotStartInitDBProcess(t *testing.T) {
 
 	err = defaultInitDatabase(binTempDir, runtimeTempDir, filepath.Join(runtimeTempDir, "data"), "Tom", "Beer", "", os.Stderr)
 
-	assert.EqualError(t, err, fmt.Sprintf("unable to init database using: %s/bin/initdb -A password -U Tom -D %s/data --pwfile=%s/pwfile",
+	assert.ErrorContains(t, err, fmt.Sprintf("unable to init database using: '%s/bin/initdb -A password -U Tom -D %s/data --pwfile=%s/pwfile'",
 		binTempDir,
 		runtimeTempDir,
 		runtimeTempDir))
@@ -61,7 +61,7 @@ func Test_defaultInitDatabase_ErrorInvalidLocaleSetting(t *testing.T) {
 
 	err = defaultInitDatabase(tempDir, tempDir, filepath.Join(tempDir, "data"), "postgres", "postgres", "en_XY", os.Stderr)
 
-	assert.EqualError(t, err, fmt.Sprintf("unable to init database using: %s/bin/initdb -A password -U postgres -D %s/data --pwfile=%s/pwfile --locale=en_XY",
+	assert.ErrorContains(t, err, fmt.Sprintf("unable to init database using: '%s/bin/initdb -A password -U postgres -D %s/data --pwfile=%s/pwfile --locale=en_XY'",
 		tempDir,
 		tempDir,
 		tempDir))

--- a/prepare_database_test.go
+++ b/prepare_database_test.go
@@ -40,7 +40,8 @@ func Test_defaultInitDatabase_ErrorWhenCannotStartInitDBProcess(t *testing.T) {
 
 	err = defaultInitDatabase(binTempDir, runtimeTempDir, filepath.Join(runtimeTempDir, "data"), "Tom", "Beer", "", os.Stderr)
 
-	assert.ErrorContains(t, err, fmt.Sprintf("unable to init database using: '%s/bin/initdb -A password -U Tom -D %s/data --pwfile=%s/pwfile'",
+	assert.NotNil(t, err)
+	assert.Contains(t, err.Error(), fmt.Sprintf("unable to init database using: '%s/bin/initdb -A password -U Tom -D %s/data --pwfile=%s/pwfile'",
 		binTempDir,
 		runtimeTempDir,
 		runtimeTempDir))
@@ -61,7 +62,8 @@ func Test_defaultInitDatabase_ErrorInvalidLocaleSetting(t *testing.T) {
 
 	err = defaultInitDatabase(tempDir, tempDir, filepath.Join(tempDir, "data"), "postgres", "postgres", "en_XY", os.Stderr)
 
-	assert.ErrorContains(t, err, fmt.Sprintf("unable to init database using: '%s/bin/initdb -A password -U postgres -D %s/data --pwfile=%s/pwfile --locale=en_XY'",
+	assert.NotNil(t, err)
+	assert.Contains(t, err.Error(), fmt.Sprintf("unable to init database using: '%s/bin/initdb -A password -U postgres -D %s/data --pwfile=%s/pwfile --locale=en_XY'",
 		tempDir,
 		tempDir,
 		tempDir))

--- a/prepare_database_test.go
+++ b/prepare_database_test.go
@@ -41,7 +41,7 @@ func Test_defaultInitDatabase_ErrorWhenCannotStartInitDBProcess(t *testing.T) {
 	err = defaultInitDatabase(binTempDir, runtimeTempDir, filepath.Join(runtimeTempDir, "data"), "Tom", "Beer", "", os.Stderr)
 
 	assert.NotNil(t, err)
-	assert.Contains(t, err.Error(), fmt.Sprintf("unable to init database using: '%s/bin/initdb -A password -U Tom -D %s/data --pwfile=%s/pwfile'",
+	assert.Contains(t, err.Error(), fmt.Sprintf("unable to init database using '%s/bin/initdb -A password -U Tom -D %s/data --pwfile=%s/pwfile'",
 		binTempDir,
 		runtimeTempDir,
 		runtimeTempDir))
@@ -63,7 +63,7 @@ func Test_defaultInitDatabase_ErrorInvalidLocaleSetting(t *testing.T) {
 	err = defaultInitDatabase(tempDir, tempDir, filepath.Join(tempDir, "data"), "postgres", "postgres", "en_XY", os.Stderr)
 
 	assert.NotNil(t, err)
-	assert.Contains(t, err.Error(), fmt.Sprintf("unable to init database using: '%s/bin/initdb -A password -U postgres -D %s/data --pwfile=%s/pwfile --locale=en_XY'",
+	assert.Contains(t, err.Error(), fmt.Sprintf("unable to init database using '%s/bin/initdb -A password -U postgres -D %s/data --pwfile=%s/pwfile --locale=en_XY'",
 		tempDir,
 		tempDir,
 		tempDir))

